code/modules/research/experimentor.dm

```diff
@@ -291,7 +291,8 @@
 				if(!T.density)
 					if(prob(EFFECT_PROB_VERYHIGH))
 						var/obj/effect/decal/cleanable/reagentdecal = new/obj/effect/decal/cleanable/greenglow(T)
-						reagentdecal.reagents.add_reagent("radium", 7)
+						if(reagentdecal)
+							reagentdecal.reagents.add_reagent("radium", 7)

```

Comment:
Rather than this you need to attempt to find an existing cleanable of this type, then if there isn't one create a new one.